es.json

```diff
@@ -1,71 +1,71 @@
+			"option-auto-pair-markdown-description": "Empareja automáticamente los símbolos para negrita y cursiva.",
+			"option-smart-indent-lists": "Listas con indentado inteligente",
+			"option-smart-indent-lists-description": "Gestiona la indentación y marcador de lista por ti.",
+			"option-fold-heading": "Colapsar cabecera",
+			"option-fold-heading-description": "Permite colapsar todo bajo una cabecera.",
+			"option-fold-indent": "Indentación de colapsado",
+			"option-fold-indent-description": "Permite colapsar una porción con el mismo indentado como, por ejemplo, listas.",
+			"option-show-line-number": "Muestra número de línea",
+			"option-show-line-number-description": "Muestra los números de línea en la canaleta.",
+			"option-use-tabs": "Usar tabulador",
+			"option-use-tabs-description": "Usa el tabulador para indentar cuando se presiona la tecla \"Tab\". Desactivar para usar espacios.",
+			"option-vim-key-bindings": "Atajos de teclado de Vim",
+			"option-vim-key-bindings-description": "Permite el uso de los atajos de teclado de Vim en el editor."
 		},
 		"file": {
-			"name": "File",
-			"option-confirm-file-deletion": "Confirm file deletion",
-			"option-confirm-file-deletion-description": "Whether you want to get prompted to confirm file deletion.",
-			"option-delete-destination": "Deleted files",
-			"option-delete-destination-description": "What to do with files after you delete them.",
-			"option-choice-system-trash": "Move to system trash",
-			"option-choice-vault-trash": "Move to Obsidian trash (.trash folder)",
-			"option-choice-permanent-delete": "Permanent delete",
-			"option-always-update-links": "Always update links when",
-			"option-always-update-links-description": "Whether to automatically update links after renaming a file or prompt.",
-			"option-new-note-location": "New note default location",
-			"option-new-note-location-description": "Where to put newly created notes. Plugin settings will override this.",
-			"option-choice-vault-root": "Vault root",
-			"option-choice-current-folder": "Same folder as the file is in",
-			"option-choice-specified-folder": "In the folder specified below",
-			"option-new-file-folder-path": "Folder to create new notes in",
-			"option-new-file-folder-path-description": "Newly created notes will appear under this folder. Plugin settings will override this.",
-			"option-link-autocompleted-format": "New link format",
-			"option-link-autocompleted-format-description": "What links to insert when auto generating internal links.",
-			"option-choice-shortest-linktext": "Shortest path when possible",
-			"option-choice-relative-path": "Relative path to file",
-			"option-choice-absolute-path": "Absolute path in vault",
-			"option-attachment-folder-path": "Attachment folder path",
-			"option-attachment-folder-path-description": "Newly created attachment files will appear here. That includes images created via drag and drop or pasting, and audio recordings."
+			"name": "Fichero",
```

Comment:
@ericaxu how do you keep track of Spanish variations? That would be "Archivo" for me

Reply:
Yeah, you're right! "Archivo" is a better fit